config/src/vespa/config/configgen/value_converter.h

```diff
@@ -7,7 +7,7 @@
 
 namespace config::internal {
 
-void requireValid(const vespalib::string & __fieldName, const ::vespalib::slime::Inspector & __inspector);
+void requireValid(vespalib::stringref __fieldName, const ::vespalib::slime::Inspector & __inspector);
```

Comment:
Related to the other comment, we should stop using double underscores here as well

Reply:
Agree, but I think that belongs in a different PR